src_c/event.c

```diff
@@ -1982,6 +2023,12 @@ pg_event_get(PyObject *self, PyObject *args, PyObject *kwargs)
 
     _pg_event_pump(dopump);
 
+    int i;
+    for (i = 0; i < NUM_KEYS; i++) {
+        held_down_keys[i] = 0;
+        released_keys[i] = 0;
+    }
+
```

Comment:
The "fancy" way of doing this (imo) would be to use memset. (https://www.man7.org/linux/man-pages/man3/memset.3.html)

I believe this would work?
`memset(held_down_keys, 0, sizeof(held_down_keys);`